src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs

```diff
@@ -9,21 +9,52 @@
 
 namespace Microsoft.PowerFx.Interpreter.Tests
 {
+    [Collection("Interpreter")]
```

Comment:
>[Collection("Interpreter")] [](http://example.com/codeflow?start=4&length=27)

What does this do?

Reply:
It groups all tests in this class with those in LanguageServerTests and have them run sequentially
[https://xunit.net/docs/running-tests-in-parallel](https://xunit.net/docs/running-tests-in-parallel)